src/app/rosetta/lib/search.ml

```diff
@@ -991,25 +991,38 @@ module Sql = struct
       User_commands.run ~logger ~offset ~limit (module Conn) query
       |> Errors.Lift.sql ~context:"Finding user commands with transaction query"
     in
+
+    (* user_command_count is a total number of user commands disregard limit & offset paramaters
```

Comment:
This `User_commands.run` API seems pretty counterintuitive. I suggest if possible we update the function `User_commands.run` to return a struct to have names on these return values at least to indicate they're not necessarily related.

Reply:
BTW, is the number of results ignoring offsets&limits actually needed in reality? I assumes this is some kind of search engine behavior telling us we have x in total, and we're on some page y(where offset&limit is actully used)?